lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -1,13 +1,13 @@
 require 'shellwords'
 require 'yaml'
 
-require 'smart_proxy_dynflow/runner/command'
+require 'smart_proxy_dynflow/runner/process_manager_command'
 require 'smart_proxy_dynflow/runner/base'
 require 'smart_proxy_dynflow/runner/parent'
 module Proxy::Ansible
   module Runner
     class AnsibleRunner < ::Proxy::Dynflow::Runner::Parent
-      include ::Proxy::Dynflow::Runner::Command
+      include ::Proxy::Dynflow::Runner::ProcessManagerCommand
       attr_reader :execution_timeout_interval, :command_pid
 
       def initialize(input, suspended_action:)
```

Comment:
I know it is not strictly related, but now it is untestable with dynflow master, can we add the id: from [`On demand`](https://github.com/theforeman/smart_proxy_dynflow/pull/105) first so we won't forget (probably separate PR)?
```suggestion
      def initialize(input, suspended_action:, id: nil)
```

Reply:
right... if I'd just read first :) it alrady implemented as #63